### PR TITLE
Create New User (POST /user)

### PR DESCRIPTION
**Pull request by hyecheol123**

### CODE REVIEW

File: src/datatypes/user/NewUserForm.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+ * Interface for NewUserForm
+ */
+export default interface NewUserForm {
+  username: string;
+  password: string;
+  admissionYear: number;
+  nameKorean: string;
+  nameEnglish?: string;
```
Comment (bsstayo) [inline src/datatypes/user/NewUserForm.ts:15]: optional maybe?

File: src/datatypes/user/NewUserForm.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Define type for new uer form
+ *
+ * @author Hyecheol (Jerry) Jang <hyecheol123@gmail.com>
+ */
+
+/**
+ * Interface for NewUserForm
+ */
+export default interface NewUserForm {
```
Comment (bsstayo) [inline src/datatypes/user/NewUserForm.ts:10]: Nickname?

File: src/datatypes/user/NewUserForm.ts (hunk 1 above)
Comment (hyecheol123) [reply src/datatypes/user/NewUserForm.ts:15]: https://www.typescriptlang.org/docs/handbook/2/objects.html

File: src/datatypes/user/NewUserForm.ts (hunk 2 above)
Comment (bsstayo) [reply src/datatypes/user/NewUserForm.ts:10]: Is it too much collecting user's school info or major or their job etc?

File: src/datatypes/user/NewUserForm.ts (hunk 1 above)
Comment (hyecheol123) [reply src/datatypes/user/NewUserForm.ts:15]: Runtime checkers are implemented separately

Comment (bsstayo) [reply src/datatypes/user/NewUserForm.ts:15]: I mean do we REALLY NEED english name

**Review by bsstayo — APPROVED** [review]
Well done!

File: src/datatypes/user/NewUserForm.ts (hunk 2 above)
Comment (hyecheol123) [reply src/datatypes/user/NewUserForm.ts:10]: As an optional field?
Nickname will not be used. Users should use their legal name.

File: src/datatypes/user/NewUserForm.ts (hunk 1 above)
Comment (hyecheol123) [reply src/datatypes/user/NewUserForm.ts:15]: That's why it is optional.

**Pull request merged**